### DISCUSSION TITLE
When using appDir, should the viewport be set?

### DISCUSSION THREAD

**Discussion by bbar** [vercel/next.js#42275] [Ideas]
Currently if we use `npx create-next-app@latest` (appDir is not enabled), the following is injected to the head:

`<meta name="viewport" content="width=device-width" />`

However, if we use `npx create-next-app@latest --experimental-app` to use the new appDir, then the viewport is not set and the app is not responsive. Should we consider adding the following, or should that be left to the developer?

`<meta name="viewport" content="width=device-width, initial-scale=1.0" />`

(It was very confusing to me not being there.)

**Comment by icyJoseph** [comment]
Ah, perhaps you should think of adding it yourself. Don't forget about charset either. 

```html
<meta charset="UTF-8">
```
